scripts/space-provider-config

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# This is a helper script to set the space provider access from config file
+# as Config value in space provider yaml.
+
+# Check if two parameters are provided
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 <config file> <provider yaml>"
+    exit 1
+fi
+
+config="$1"
+provider="$2"
+
+# Check if config file exists
+if [ ! -f "$config" ]; then
```

Comment:
https://www.gnu.org/software/bash/manual/bash.html#Bash-Conditional-Expressions says you could test `-r` here

Reply:
Changed